NadekoBot.Core/_Extensions/IMessageChannelExtensions.cs

```diff
@@ -62,12 +62,12 @@ public static Task<IUserMessage> SendTableAsync<T>(this IMessageChannel ch, stri
         public static Task SendPaginatedConfirmAsync(this ICommandContext ctx,
             int currentPage, Func<int, EmbedBuilder> pageFunc, int totalElements,
             int itemsPerPage, bool addPaginatedFooter = true)
-            => ctx.SendPaginatedConfirmAsync(currentPage,
+            => ctx.SendPaginatedConfirmAsync2(currentPage,
```

Comment:
Can you figure out why the overloading didn't work and try to resolve it? This could make it harder to maintain the fork in the future.

Reply:
idk why it didnt work and i tried to resolve for hours

Reply:
Ok I'll take a look